service/service.py

```diff
@@ -132,13 +132,12 @@ def internal_server_error(error):
 
 # Define the Wishlist-Product model so that the docs reflect what can be sent
 wishlist_product_model = api.model('WishlistProduct', {
-    'wishlist_id': fields.Integer(readOnly=True,
-                                  description='Wishlist unique ID'),
+        'wishlist_id': fields.Integer(readOnly=True,
```

Comment:
please fix the indentation here

Reply:
I had to modify it to increase the score, it was wrong.

Reply:
ok. Thank you